src/utils/disable-download.ts

```diff
@@ -0,0 +1,9 @@
+import semver from 'semver';
+
+export function disableDownload(version: string) {
+  return (
+    process.platform === 'darwin' &&
+    process.arch === 'arm64' &&
+    semver.lt(version, '11.0.0')
+  );
```

Comment:
To improve the stability of subsequent unit tests, please change to:

```suggestion
  return (
    !process.env.JEST &&
    process.platform === 'darwin' &&
    process.arch === 'arm64' &&
    semver.lt(version, '11.0.0')
  );
```

See:
* https://github.com/electron/fiddle/pull/1022
* https://github.com/electron/fiddle/pull/992